src/library-authoring/common/specs/OrganizationDropdown.spec.jsx

```diff
@@ -29,16 +30,21 @@ const props = {
   errorCode: null,
   readOnly: false,
 };
+const InjectedOrganizationDropdown = injectIntl(OrganizationDropdown);
 
 describe('common/OrganizationDropdown.jsx', () => {
   it('renders component without error', () => {
-    mount(<OrganizationDropdown {...props} />);
+    mount(<IntlProvider locale="en">
```

Comment:
Please clarify me, why don't you use `ctxMount` here?
That includes `IntlProvider locale="en"` already

Reply:
Thanks, I completely forget about it.